docs/_docs/versions.md

```diff
@@ -7,112 +7,161 @@ redirect_from: /docs/
+
+In the CLI, running `zapier promote [version]` will make the specified version number the new Public version. [Read more about that here](https://platform.zapier.com/cli_docs/docs#promoting-an-app-version).
+
+In the Visual Builder, head to **Manage** then **Versions**, and select the **gear icon** next to the version you’d like to make public. In the menu that appears, select **Promote**.
+
+![Selecting 'Promote' on the Versions page](https://cdn.zappy.app/dd4fccc5acb2beb2194ca448871c0d4d.png)
+
+If you’re working with an integration that’s currently private, the _Promote_ option won’t appear. Instead, make sure users you’d like to test or move to the new integration version are invited to that version, and encourage them to try it out.
+
+### What happens when a version is promoted?
+
+* Zap templates are updated to use the new Public version of your app
+* Any new triggers or actions will appear on your integration’s public app page
+* When a user selects your integration in a new Zap, they’ll use the promoted version by default
+
+## Migrating users
 
 Once you’ve added all the changes needed to your integration, you can start rolling it out to users to update their existing Zaps and let users make new Zaps with your new integration features.
 
-If you made a patch version with only minor, non-breaking changes, you can migrate existing users and Zaps to your new integration version. Select the _Migrate_ option on your new integration version, then choose which older version of your integration to move users from, and the new version to move them to. Finally, choose what percentage of users to migrate. 
+If you made a new version with only minor, non-breaking changes, you can migrate existing users and Zaps to your new integration version. On the **Versions** page, select the **Migrate** option on your new integration version.
+
+![Selecting the 'Migrate' option on the Versions page](https://cdn.zappy.app/ee93e158cfe97f48a2cdc7b8881df405.png)
+
+That will reveal a screen that will allow you to move users between two versions. Select the _From_ and _To_ versions, then select **Migrate** to begin moving users to the selected version.
+
+![After selecting 'Migrate', the Migration window allows you to select 'From' and 'To' versions](https://cdn.zappy.app/5d8d487544914ddaa1c72037b331aad1.png)
+
+When migrating users, you are also migrating their Zaps. This means that any active Zaps using the _From_ version will begin using the _To_ version as soon as the migration is complete. This makes it a great solution for Patch and Minor updates, but _not_ a recommended option for Major updates. For Major updates, review our section on Deprecation instead.
+
+In some cases, it may be more useful to migrate only a portion of the current users, to ensure that the minor change did not accidentally introduce any unforeseen issues. You can do that using the **Percentage** option in the Migration window, or select **Email** to migrate only one user at a time.
```

Comment:
We might want to add a warning about the "migrate by email"  tool as it doesn't always migrate a user only.

```
> **WARNING:** When you specify a user email that belongs to a team, "migrating by email" will migrate that user **and all their team members**! If your intention is to migrate a user, make sure to use a user account that is not in a team account.
```